### DISCUSSION TITLE
i can't receive two factor authentication code

### DISCUSSION THREAD

**Discussion by pr0jhayson** [community/community#128024] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/76271890/99dfa7aa-4bda-4163-910f-d8b6fc7b57cd)
![image](https://github.com/community/community/assets/76271890/c7d06f0c-73c4-4417-8abd-d0cd38c605d5)
help!! i can't receive sms.. i also tried to find live customer support, but i couldn't find it

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @pr0jhayson ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by davevad93** [comment]
Hi @pr0jhayson ,

I think you should contact support by opening a ticket [here](https://support.github.com/contact/account), select `Two-factor authentication recovery` and explain your issue. Note that it may take a while for them to give an answer, so be patient.
